src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      for (const key in action.extraData) {
+        state[key] = action.extraData[key];
+      }
```

Comment:
This block is better to rewrite with Object.assign.

Reply:
мне линтер выдает тогда ошибку, что нельзя присваивать state новое значение

Reply:
You can do it without `=`
```suggestion
      Object.assign(state, action.extraData);
```